CalibTracker/SiStripLorentzAngle/src/LA_Filler.cc

```diff
@@ -85,11 +86,11 @@ granularity(const SiStripDetId detid, const float tthetaL, const Long64_t TFE_in
 }
 
 std::string LA_Filler_Fitter::
-subdetLabel(const SiStripDetId detid) { return detid.subDetector()==SiStripDetId::TOB? "TOB" : "TIB";}
+subdetLabel(const SiStripDetId detid) { return detid.subDetector()==SiStripDetId::TOB ? "TOB" : "TIB";}
 std::string LA_Filler_Fitter::
 moduleLabel(const SiStripDetId detid) { return subdetLabel(detid) + "_module"+boost::lexical_cast<std::string>(detid());}
 std::string LA_Filler_Fitter::
 layerLabel(const SiStripDetId detid) {
-  unsigned layer = detid.subDetector() == SiStripDetId::TOB ? TOBDetId(detid()).layer() : TIBDetId(detid()).layer();
+  unsigned layer = detid.subDetector() == getTIBOrTOBLayer(detid);
```

Comment:
should be `unsigned layer = getTIBOrTOBLayer(detid);` (the current code will compare layer do subdetector numbers)